File: LUYENCODE/PROBLEMS/DK04 - Làm tròn số/DK04.py
a = float(input()) 

if a < 0:
    if abs(a) - abs(int(a)) >= 0.5:
        print(int(a) - 1) 
    else:
        if int(a) == 0:
            print(0)  
        else:
            print(int(a)) 
else:
    if a - int(a) >= 0.5:
        print(int(a) + 1)  
    else:
        print(int(a)) 

File: LUYENCODE/PROBLEMS/DK04 - Làm tròn số/DK04.cs
using System;
namespace DK04
{
    public class Program
    {
        static void Main(string[]args)
        {
            double a;
            a = double.Parse(Console.ReadLine());  
            
            if (a < 0)
            {
                if (Math.Abs(a) - Math.Abs((int)a) >= 0.5)
                    Console.WriteLine((int)a - 1); 
                else
                {
                    if ((int)a == 0)
                        Console.WriteLine(0); 
                    else 
                        Console.WriteLine((int)a);  
                }
            }
            else
            {
                if (a - (int)a >= 0.5)
                    Console.WriteLine((int)a + 1);  
                else
                    Console.WriteLine((int)a);
            }
        }
    }
}
